### ISSUE TITLE
[Blue Burst] Fail to create EP1 Ultimate Mode room

### ISSUE THREAD

**Issue by Sazonde99** [fuzziqersoftware/newserv#121]
![image](https://github.com/fuzziqersoftware/newserv/assets/123838641/9670e57a-8302-4d2c-bbbc-109789a3cf68)

However, the below do work:
EP1: Normal/Hard/Very Hard
EP2: Normal/Hard/Very Hard/Ultimate
EP4: Normal/Hard/Very Hard/Ultimate

Probably since EP1 Ultimate is an outlier with the different areas, enemies, boss models.

**Comment by nolrinale** [comment]
Attaching the log snippet of this particular problem it still happens in the latest builds:

```
I 453 2023-10-31 14:52:47 - [StaticGameData] (Entry 67F) [Map::Enemy VOL_OPT_2 flags=00 last_hit_by_client_id=0]
I 453 2023-10-31 14:52:47 - [C-6] [Map/14] Using variations 0, 0
I 453 2023-10-31 14:52:47 - [StaticGameData] ./data/map_boss04e.dat missing from BB patch tree
I 453 2023-10-31 14:52:47 - [StaticGameData] Loaded map_boss04e.dat from data.gsl in BB patch tree
[0000] C8000000FFFF00000E00221D0000010001000000005C2E410000A040B45C21C3000000000000000000000000000000000000000000000000000000000000000000000000B061FA8C
I 453 2023-10-31 14:52:47 - [C-6] [Map/14] Loaded map_boss04e.dat (513 entries)
I 453 2023-10-31 14:52:47 - [StaticGameData] (Entry 680) [Map::Enemy DARK_FALZ_3 flags=00 last_hit_by_client_id=0]
I 453 2023-10-31 14:52:47 - [C-6] [Map/14] Failed to load map_boss04e.dat: invalid enemy type
W 453 2023-10-31 14:52:47 - [Server] Error processing client command: no maps loaded for area 14
I 453 2023-10-31 14:52:47 - [Server] Client disconnected: C-6 on fd 33
I 453 2023-10-31 14:52:47 - [C-6] Deleted

```

It seems to be the server doesn't like the `map_boss04e.dat` served from the data.gsl (Tethealla one) or cannot parse it correctly for some reason

**Comment by fuzziqersoftware** [comment]
This has been fixed by 96e47819.
